packages/logs/src/domain/logsCollection/runtimeError/runtimeErrorCollection.spec.ts

```diff
@@ -8,43 +7,60 @@ import { LifeCycle, LifeCycleEventType } from '../../lifeCycle'
 import { startRuntimeErrorCollection } from './runtimeErrorCollection'
 
 describe('runtime error collection', () => {
-  let rawErrorObservable: Observable<RawError>
+  const configuration = { forwardErrorsToLogs: true } as LogsConfiguration
   let lifeCycle: LifeCycle
   let stopRuntimeErrorCollection: () => void
   let rawLogsEvents: Array<RawLogsEventCollectedData<RawRuntimeLogsEvent>>
+  let onErrorSpy: jasmine.Spy
+  let originalOnErrorHandler: OnErrorEventHandler
 
   beforeEach(() => {
+    originalOnErrorHandler = window.onerror
+    onErrorSpy = jasmine.createSpy()
+    window.onerror = onErrorSpy
     rawLogsEvents = []
-    rawErrorObservable = new Observable<RawError>()
     lifeCycle = new LifeCycle()
     lifeCycle.subscribe(LifeCycleEventType.RAW_LOG_COLLECTED, (rawLogsEvent) =>
       rawLogsEvents.push(rawLogsEvent as RawLogsEventCollectedData<RawRuntimeLogsEvent>)
     )
-    ;({ stop: stopRuntimeErrorCollection } = startRuntimeErrorCollection(
-      {} as LogsConfiguration,
-      lifeCycle,
-      rawErrorObservable
-    ))
   })
 
   afterEach(() => {
     stopRuntimeErrorCollection()
+    window.onerror = originalOnErrorHandler
   })
 
-  it('should send runtime errors', () => {
-    rawErrorObservable.notify({
-      message: 'error!',
-      source: ErrorSource.SOURCE,
-      startClocks: { relative: 1234 as RelativeTime, timeStamp: 123456789 as TimeStamp },
-      type: 'Error',
+  it('should send runtime errors', (done) => {
```

Comment:
I updated the test to really trigger a runtime error instead of using the observable, because I found it more secure.